charts/service-binding-operator/values.yaml

```diff
@@ -1,8 +1,8 @@
 image:
   pullPolicy: IfNotPresent
-  repository: quay.io/redhat-developer
+  repository: quay.io/redhat-developer/servicebinding-operator@sha256:67c2a2502f59fac1e7ded9ed19b59bbd4e50f5559a13978a87ecd2283b81e067
```

Comment:
Technically, with your change, the value here is actually an image ref, not a repository.

IMHO, it would be confusing to set the image ref into a property called `image.repository`.

If you need to set the FQN of the image ref (e.g. in the `helm install` command) as a single property, you should rather keep the `image.repository` and introduce a new property such as `image.image` and have the default value be constructed as `{{ Values.image.repository }}/servicebinding-operator@sha256:67c2a2502f59fac1e7ded9ed19b59bbd4e50f5559a13978a87ecd2283b81e067` or similarly, and then you can set the `image.image` as a single value.

Reply:
I applied your suggestion